BFH_USZ_PICC/BFH_USZ_PICC.iOS/DependencyServices/ReminderNotification.cs

```diff
@@ -13,31 +13,34 @@ public class ReminderNotification : Interfaces.IReminderNotification
         string Body;
         readonly double DAILYSECONDS = 84400;
 
-        public void AddNotification(DateTimeOffset maintenanceReminderStartDateTime, int intervalInDays, int maintenanceReminderRepetition, string title, string body)
+        public void AddNotification(DateTimeOffset maintenanceReminderStartDateTime, int intervalInDays, int maintenanceReminderRepetition, string title, string body, bool isUnlimited)
         {   
             UILocalNotification noti = new UILocalNotification();                      
             noti.AlertTitle = title;
             noti.AlertBody = body;
-                                              
-            NSDate startDate = DateTimeToNSDate(maintenanceReminderStartDateTime);
+            noti.AlertLaunchImage = "Icon.png";
+
+            NSDate startDateTime = DateTimeToNSDate(maintenanceReminderStartDateTime);
             
             double reminderIntervalInSeconds = DAILYSECONDS * intervalInDays;
 
-            int countReminderRepetitions = 0;
-            while (countReminderRepetitions <= maintenanceReminderRepetition)
+            // Checks if the user wants a to set a repetition limit or if he wants an unlimited reminder. In this case, the 500 notifications will be generated.
+            if (isUnlimited)
             {
-                // noti.FireDate = startDate.AddSeconds(reminderIntervalInSeconds * countReminderRepetitions);
-                noti.FireDate = startDate.AddSeconds(countReminderRepetitions * 20);
-                UIApplication.SharedApplication.ScheduleLocalNotification(noti);
+                int REPETITIONFOR200Times = 200;
```

Comment:
Same with the naming... btw is it ideal to define this const in each platform? I think not.